yt/visualization/fits_image.py

```diff
@@ -888,11 +888,12 @@ def construct_image(ds, axis, data_source, center, image_res, width, length_unit
         else:
             frb = data_source.to_frb(width[0], (nx, ny), center=center, height=width[1])
     elif isinstance(data_source, ParticleAxisAlignedDummyDataSource):
+        axes = axis_wcs[axis]
```

Comment:
seems like the right fix, although I can imagine it wouldn't be so obvious to me if I read this in the code first without the additional context you provided in the report. Can you add a short description for `axis_wcs` close to its definition while you're at it please ?

Reply:
done

Reply:
thanks !